package.json

```diff
@@ -54,6 +54,7 @@
     "react-eventbrite-popup-checkout": "^1.0.5",
     "react-hook-form": "^5.7.2",
     "react-image-file-resizer": "^0.3.8",
+    "react-markdown": "^6.0.2",
```

Comment:
Looks like you haven't included the `package-lock.json` updates as part of this PR yet?

Reply:
now included